projects/examples/src/app/app.component.ts

```diff
@@ -25,4 +26,11 @@ export class AppComponent {
         title: entry.displayName,
         path: entry.urlSegment,
     }));
+
+    constructor(router: Router) {
```

Comment:
Please add an explanation as to why we're doing it here and create an issue to address it later

Reply:
Issue: https://jira.eng.vmware.com/browse/VDUCC-72